AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSDBDocumentStore.m

```diff
@@ -53,65 +55,71 @@ - (instancetype)init {
 
 #pragma mark - Table Management
 
-- (BOOL)upsertWithPartition:(NSString *)partition
-            documentWrapper:(MSDocumentWrapper *)documentWrapper
-                  operation:(NSString *_Nullable)operation
-                    options:(MSBaseOptions *)options {
+- (BOOL)createUserStorageWithAccountId:(NSString *)accountId {
+
+  // Create table based on the schema.
+  return [self.dbStorage createTable:[NSString stringWithFormat:kMSUserDocumentTableNameFormat, accountId]
```

Comment:
use `[MSDBDocumentStore tableNameForPartition:accountId];`

Reply:
Will fix in a later PR